lib/common.sh

```diff
@@ -338,16 +338,16 @@ determineTool() {
             warn "For more details see: https://devcenter.heroku.com/articles/go-apps-with-modules#build-configuration"
             warn ""
         fi
-        if ! <"${DataJSON}" jq  -e '.Go.SupportsModuleExperiment | any(. == "'${ver}'")' &> /dev/null; then
+        if ! echo "\"${ver}\"" | jq -e 'split(".") | map(gsub("go";"")) | map(tonumber) | .[0] >= 1 and .[1] >= 11' &> /dev/null; then
```

Comment:
A comment would be very helpful here. This line is very difficult to parse

```
# if using 1.11 or older
```

`.[0] >= 1 and .[1] >= 11` is particularly confusing because it looks like it's testing whether that is true instead of catching when it fails. Wrapping this into a `goVersionSupportsModules` function might make this easier for others.

Reply:
done.